modules/minimum-required-permissions-ipi-gcp-provided-sas.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="minimum-required-permissions-ipi-gcp-provided-sas_{context}"]
+= Required GCP permissions for user-provided service accounts
+
+When you are installing	a cluster, the compute and control plane nodes require their own service accounts.
+The default scenario occurs when the user does not provide a service account email address for the control plane or compute nodes in the `install-config.yaml` file.
+In this case the installation program will create a service account for the control plane and compute nodes, requiring additional permissions.
```

Comment:
By "requiring additional permissions", do you mean the permissions listed in the new section "Required GCP permissions for user-provided service accounts"? Or, could you please tell what exactly you mean? Thanks!

Reply:
Yes, that is what was intended

Reply:
I misspoke. What I meant by "requiring additional permissions" was the issue raised in https://issues.redhat.com/browse/OCPSTRAT-1294, where GCP raises a security alert because the installer-created service account requires the "Service Account Admin" and "Service Account User" roles. I was referring to these roles as "additional permissions".